bcda/client/bluebutton_test.go

```diff
@@ -248,6 +249,30 @@ func (s *BBRequestTestSuite) TestGetExplanationOfBenefit_500() {
 	assert.Nil(s.T(), e)
 }
 
+func (s *BBRequestTestSuite) TestGetClaim() {
+	e, err := s.bbClient.GetClaim("1234567890hashed", "543210", "A0000", "", now, client.ClaimsWindow{})
+	assert.Nil(s.T(), err)
+	assert.Equal(s.T(), 1, len(e.Entries))
+}
+
+func (s *BBRequestTestSuite) TestGetClaim_500() {
+	e, err := s.bbClient.GetClaim("1234567890hashed", "543210", "A0000", "", now, client.ClaimsWindow{})
+	assert.Regexp(s.T(), `blue button request failed \d+ time\(s\) failed to get bundle response`, err.Error())
+	assert.Nil(s.T(), e)
+}
+
+func (s *BBRequestTestSuite) TestGetClaimResponse() {
+	e, err := s.bbClient.GetClaimResponse("1234567890hashed", "543210", "A0000", "", now, client.ClaimsWindow{})
+	assert.Nil(s.T(), err)
+	assert.Equal(s.T(), 1, len(e.Entries))
+}
+
+func (s *BBRequestTestSuite) TestGetClaimResponse_500() {
+	e, err := s.bbClient.GetClaimResponse("1234567890hashed", "543210", "A0000", "", now, client.ClaimsWindow{})
+	assert.Regexp(s.T(), `blue button request failed \d+ time\(s\) failed to get bundle response`, err.Error())
+	assert.Nil(s.T(), e)
+}
```

Comment:
There are some redundancies in this test suite as a whole, but (while not super necessary) it may be worth adding to the table driven test below. See line 322.

Reply:
Agreed, the table driven tests below are a bit more thorough and it would be beneficial to add Claim and ClaimResponse tests there